apps/portals/src/configurations/elportal/synapseConfigs/studies.ts

```diff
@@ -30,9 +30,10 @@ export const studyCardConfiguration: CardConfiguration = {
     },
     {
       isMarkdown: false,
-      baseURL: 'Explore/Projects/DetailsPage',
-      URLColumnName: 'shortName',
-      matchColumnName: 'projectShortName',
+      matchColumnName: 'projectFullName',
+      overrideLinkURLColumnName: 'project',
+      overrideLinkURLColumnTransform: (shortName) =>
+        `/Explore/Projects/DetailsPage?shortName=${shortName}`,
```

Comment:
Use a new feature - the displayed value is retrieved from the `projectFullName` column, the `href` is derived from the `project` column, and the actual href value a transform of the value found in the `project` column